docs/UserGuide.md

```diff
@@ -373,9 +448,17 @@ Exits the program.
 
 Format: `exit`
 
+--------------------------------------------------------------------------------------------------------------------
+
+## Miscellaneous Features 
+
+Here you can find miscellaneous features. These are only for advanced users as they have the potential to cause
+HustleBook to wipe your data.
+
 ### Saving the data
 
 HustleBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+The data is saved in a `json` file found in `data` folder. 
```

Comment:
It would be good to include: 
`found in a folder named data at the location HustleBook is saved in`